### DISCUSSION TITLE
Certificate is not valid on custom domain

### DISCUSSION THREAD

**Discussion by thebigredgeek** [vercel/community#3207] [Help]
### Problem Description

See pics below:

![Screenshot 2023-07-02 at 7 05 31 PM](https://github.com/vercel/vercel/assets/2799276/c4b65582-5d85-4715-b284-d01b2009dab3)
![Screenshot 2023-07-02 at 7 05 42 PM](https://github.com/vercel/vercel/assets/2799276/449e53b8-6bd2-40f6-ae80-f0216e2bb9fa)

Any ideas?  Vercel generated the certs, the certs seem to be issued from Let's Encrypt.  Is there a delay?

**Comment by amyegan** [comment]
Hi @thebigredgeek. I don't see any cert problems when I visit the site now. Are you still having this problem?
